picoharpApp/src/picopeaks.c

```diff
@@ -423,27 +424,27 @@ bool pico_measure(struct pico_data *self, int delay)
 
     self->overflow = 0;
 
-    PICO_CHECK(PH_ClearHistMem(self->device, BLOCK));
-    PICO_CHECK(PH_StartMeas(self->device, delay));
+    PICO_CHECK(PH330_ClearHistMem(self->device));
+    PICO_CHECK(PH330_StartMeas(self->device, delay));
     self->current_time = delay;
 
     while (true)
     {
         int done = 0;
-        PICO_CHECK(PH_CTCStatus(self->device, &done));
+        PICO_CHECK(PH330_CTCStatus(self->device, &done));
         if(done)
             break;
         usleep(10000);  // 10 ms poll
     }
 
     int Flags = 0;
-    PICO_CHECK(PH_StopMeas(self->device));
-    PICO_CHECK(PH_GetHistogram(self->device, self->countsbuffer, BLOCK));
-    PICO_CHECK(PH_GetFlags(self->device, &Flags));
+    PICO_CHECK(PH330_StopMeas(self->device));
+    PICO_CHECK(PH330_GetHistogram(self->device, self->countsbuffer, BLOCK));    // only for channel 1?
```

Comment:
Don't leave unanswered questions in the code if at all possible.  I don't understand this question, so ask it separately in a more detailed way.

Reply:
We're always interested in counting the delay from channel 1 (machine revolution clock) to channel 2 (photon event from beam), so I presume we're only interested in one histogram ... I didn't know there was a second histogram available!  Is the code doing it wrong, is there a general fix to be made here?

Reply:
Thank you very much, I understand. The new PH330 has a separate Sync Channel and up to 4 Input Channel. So I have to change the code for Channel 1 and 2 to be now Sync and Channel 1.